.github/workflows/workflow.yaml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - master
+      - validation
```

Comment:
I think we can remove this before merging, right?

Reply:
Yes, for sure